models/object.py

```diff
@@ -42,7 +41,7 @@ class DraggableObject(QWidget):
     newGeometry = Signal(QRect)
 
     # Parent should be called editor, its always the editor
-    def __init__(self, parent, p, cWidget,type):
```

Comment:
this can be found in the constructor, its cleaner and doesnt leave room for someone to say that its a type that its not